### ISSUE TITLE
Hang on by step "estimator.fit(data_location)" in tensorflow_bring_your_own.ipynb 

### ISSUE THREAD

**Issue by liangyimingcom** [jackie930/steward-sentiment-anlysis-bot#2]
**hello, Jackie
求助。
停止在这一步 "estimator.fit(data_location)" in tensorflow_bring_your_own.ipynb ，最后不动的代码如下：**

b'2021-05-06 09:11:20,722 - tensorflow - WARNING - From run_classifier.py:742: The name tf.trainable_variables is 
deprecated. Please use tf.compat.v1.trainable_variables instead.\n'
b'\n'
b'2021-05-06 09:11:20,728 - tensorflow - WARNING - From run_classifier.py:756: The name tf.train.init_from_checkpoint is deprecated. Please use tf.compat.v1.train.init_from_checkpoint instead.\n'
b'\n'
b'2021-05-06 09:11:21,464 - tensorflow - INFO - **** Trainable Variables ****\n'
b'2021-05-06 09:11:21,465 - tensorflow - INFO -   name = bert/embeddings/word_embeddings:0, shape = (21128, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,465 - tensorflow - INFO -   name = bert/embeddings/token_type_embeddings:0, shape = (2, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,465 - tensorflow - INFO -   name = bert/embeddings/position_embeddings:0, shape = (512, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,465 - tensorflow - INFO -   name = bert/embeddings/LayerNorm/beta:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,465 - tensorflow - INFO -   name = bert/embeddings/LayerNorm/gamma:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,466 - tensorflow - INFO -   name = bert/encoder/layer_0/attention/self/query/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,466 - tensorflow - INFO -   name = bert/encoder/layer_0/attention/self/query/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,466 - tensorflow - INFO -   name = bert/encoder/layer_0/attention/self/key/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,466 - tensorflow - INFO -   name = bert/encoder/layer_0/attention/self/key/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,466 - tensorflow - INFO -   name = bert/encoder/layer_0/attention/self/value/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,466 - tensorflow - INFO -   name = bert/encoder/layer_0/attention/self/value/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,467 - tensorflow - INFO -   name = bert/encoder/layer_0/attention/output/dense/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,467 - tensorflow - INFO -   name = bert/encoder/layer_0/attention/output/dense/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,467 - tensorflow - INFO -   name = bert/encoder/layer_0/attention/output/LayerNorm/beta:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,467 - tensorflow - INFO -   name = bert/encoder/layer_0/attention/output/LayerNorm/gamma:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,467 - tensorflow - INFO -   name = bert/encoder/layer_0/intermediate/dense/kernel:0, shape = (768, 3072), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,468 - tensorflow - INFO -   name = bert/encoder/layer_0/intermediate/dense/bias:0, shape = (3072,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,468 - tensorflow - INFO -   name = bert/encoder/layer_0/output/dense/kernel:0, shape = (3072, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,468 - tensorflow - INFO -   name = bert/encoder/layer_0/output/dense/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,468 - tensorflow - INFO -   name = bert/encoder/layer_0/output/LayerNorm/beta:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,468 - tensorflow - INFO -   name = bert/encoder/layer_0/output/LayerNorm/gamma:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,468 - tensorflow - INFO -   name = bert/encoder/layer_1/attention/self/query/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,469 - tensorflow - INFO -   name = bert/encoder/layer_1/attention/self/query/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,469 - tensorflow - INFO -   name = bert/encoder/layer_1/attention/self/key/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,469 - tensorflow - INFO -   name = bert/encoder/layer_1/attention/self/key/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,469 - tensorflow - INFO -   name = bert/encoder/layer_1/attention/self/value/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,469 - tensorflow - INFO -   name = bert/encoder/layer_1/attention/self/value/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,469 - tensorflow - INFO -   name = bert/encoder/layer_1/attention/output/dense/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,470 - tensorflow - INFO -   name = bert/encoder/layer_1/attention/output/dense/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,470 - tensorflow - INFO -   name = bert/encoder/layer_1/attention/output/LayerNorm/beta:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,470 - tensorflow - INFO -   name = bert/encoder/layer_1/attention/output/LayerNorm/gamma:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,470 - tensorflow - INFO -   name = bert/encoder/layer_1/intermediate/dense/kernel:0, shape = (768, 3072), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,470 - tensorflow - INFO -   name = bert/encoder/layer_1/intermediate/dense/bias:0, shape = (3072,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,470 - tensorflow - INFO -   name = bert/encoder/layer_1/output/dense/kernel:0, shape = (3072, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,471 - tensorflow - INFO -   name = bert/encoder/layer_1/output/dense/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,471 - tensorflow - INFO -   name = bert/encoder/layer_1/output/LayerNorm/beta:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,471 - tensorflow - INFO -   name = bert/encoder/layer_1/output/LayerNorm/gamma:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,471 - tensorflow - INFO -   name = bert/encoder/layer_2/attention/self/query/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,471 - tensorflow - INFO -   name = bert/encoder/layer_2/attention/self/query/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-06 09:11:21,471 - tensorflow - INFO -   name = bert/encoder/layer_2/attention/self/key/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'

**Comment by liangyimingcom** [comment]
Amazon SageMaker -> Training jobs -> bert-tf-sentiment-analysis-2021-05-06-09-03-55-247 这个任务运行好几个小时也没结束，就是hang住了，，，😭

**Comment by jackie930** [comment]
> Amazon SageMaker -> Training jobs -> bert-tf-sentiment-analysis-2021-05-06-09-03-55-247 这个任务运行好几个小时也没结束，就是hang住了，，，😭

hello,日志 有打印loss出来吗?

**Comment by hlmiao** [comment]
Hi Jackie,
I experienced same issue, it was suspended on train mode stage as follow, how print the loss during TensorFlow training?

b'2021-05-09 09:38:56,093 - tensorflow - INFO -   name = bert/encoder/layer_2/attention/self/query/bias:0, shape = (768,), *INIT_FROM_CKPT*\n'
b'2021-05-09 09:38:56,093 - tensorflow - INFO -   name = bert/encoder/layer_2/attention/self/key/kernel:0, shape = (768, 768), *INIT_FROM_CKPT*\n'
